### ISSUE TITLE
Ballerina VSCode Plugin not started after installation

### ISSUE THREAD

**Issue by nadeeshaan** [ballerina-platform/ballerina-lang#17386]
**Description:**
Plugin fails to detect the BALLERINA_HOME and stop starting the plugin

Workaround:
Open BALLERINA_HOME/lib/tools/lang-server/launcher/language-server-launcher.sh
Replace `JAVA_HOME="$BALLERINA_HOME/bre/lib/jdk8u202-b08-jre"` with the following
```
DIR="$(cd "$(dirname "$0")" && pwd)"
JAVA_HOME="$DIR/../../../../bre/lib/jdk8u202-b08-jre"
```

**Affected Versions:**
1,0.0-alpha

**Comment by kaviththiranga** [comment]
This was handled from IDE plugin's end for alpha.

VSCode fix: https://github.com/kaviththiranga/ballerina-1/commit/d3b6401c0e8eeb9dab6b8c8a1b958d6eac8101cb

@NipunaRanasinghe Please reference Idea and close.

**Comment by kaviththiranga** [comment]
Proper fix is 4ccd594bddf78ee8ef66e6bd145bda193d5d932a and will be included in beta
